product_docs/docs/pem/9/upgrading/upgrading_pem_installation/upgrading_pem_installation_linux_rpm.mdx

```diff
@@ -8,105 +8,102 @@ redirects:
 
-```shell
-yum install edb-as<X>-server-libs
-```
+### On a Debian or Ubuntu host
 
-To upgrade PostgreSQL libs:
+You can use the `apt-get` package manager to upgrade the installed version of the PEM server on supported versions of Debian or Ubuntu:
 
 ```shell
-yum install postgresql<X>-libs
+apt-get upgrade edb-pem
 ```
 
-Where `<X>` is the PostgreSQL or EDB Postgres Advanced Server version whose `libs` package you want to install.
+After upgrading the PEM server with `apt-get`, you need to configure the PEM server. For detailed information, see [Configuring the PEM server](#configuring-the-pem-server).
 
-### Prerequisites to upgrade a PEM installation on a Debian or Ubuntu host
+### On a SLES host
 
-To upgrade packages on a Debian or Ubuntu host:
+You can use the zypper package manager to upgrade the installed version of the PEM Server on supported versions of a SLES host:
 
 ```shell
-apt-get update
-apt-get upgrade
+zypper update edb-pem
 ```
 
-To upgrade Advanced Server libs:
+After upgrading the PEM server using zypper, you need to configure the PEM server. For detailed information, see [Configuring the PEM server](#configuring-the-pem-server).
 
-```shell
-apt-get install edb-as<X>-server-libs
-```
+!!! Note
+    If you upgrade the PEM backend database server and the PEM server, update the `PG_INSTALL_PATH` and `DB_UNIT_FILE` parameters pointing to the new version in the `/usr/edb/pem/share/.install-config` file before you run the configure script.
+
+## Configuring the PEM server
 
-To upgrade PostgreSQL libs:
+After upgrading the PEM server, you can use the following command to configure the PEM server:
 
 ```shell
-apt-get install postgresql<X>-libs
+/usr/edb/pem/bin/configure-pem-server.sh
 ```
 
-Where `<X>` is the PostgreSQL or EDB Postgres Advanced Server version whose `libs` package you want to install.
+The configure script uses the values from the old PEM server configuration file while running the script.
 
-### Prerequisites to upgrade a PEM installation on a SLES host
+For detailed information, see [Configuring the PEM server on Linux platforms](/pem/latest/installing/configuring_the_pem_server_on_linux/).
 
-To upgrade packages on a SLES host:
+!!! Note
+    - From PEM version 7.11 and later, the configure script requires a superuser password only after the upgrade process.
 
-```shell
-zypper update
-zypper upgrade
-```
+    - If your configure script gets stuck, then check if any of the PEM agents are still running:
 
-To upgrade EDB Postgres Advanced Server libs:
+      ```sql
+      SELECT agent_id FROM pem.agent_config WHERE param='alert_threads' AND value > 0;
+      ```
 
-```shell
-zypper install edb-as<x>-server-libs
-```
+      Stop the running agents and re-run the configure script.
 
-To upgrade PostgreSQL libs:
+      If the problem persists, then kill the process of all the running PEM agents using the query:
```

Comment:
It should be:-
 If the problem persists, then run the below query that will terminate the alert processing processes which got stucked.